.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "_includes/babel"]
```

Comment:
We'll need to add some instructions to the readme or add that automatically in make serve/npm start?